### DISCUSSION TITLE
Approve button not so obvious

### DISCUSSION THREAD

**Discussion by jessypouliot98** [community/community#73266] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I believe the "approve" button should be a bit more obvious. I would expect this to be either with the PR details down at the bottom with the pipeline details, reviewers, etc. or at the top right near the list of reviewers.

Except it is not. It's located in the 4th tab ("files changed") under the "review changes" button with the "Approve" item selected.

I don't necessarily need to write a message or go back the the files changed tab to approve if the review was already done. Approving should be closer and on the main page of a PR

**Comment by nvaytet** [comment]
I understand the wish to force people to look at the changes before they approve, but sometimes, if the changes are large, the changes tab can be slow to load.
Once you've visited it 3-4 time because of iterations, it gets quite painful.

It would be very useful if an approve button would appear on the first tab, after you've submitted at least one review.

**Comment by ewrogers** [comment]
This is still very much a thing. Nearly *everyone* from *any* background whether it's a dev, project manager, designer, etc has no clue where the approval button is.

I understand you want to avoid "rubber stamping" but a terrible UI design isn't really the solution. You should at least put some kind of information text indicating **where** it is.

**Comment by ihorml** [comment]
Even reviewing the PR's and leaving the comments didn't help me to figure out where this button is without my teammates. 

UX really sucks guys

**Comment by Saleh-Mohammed-Alabidi** [comment]


**Reply by ihorml** [reply]
Get a fuck out of this room, ChatGPT, while adults are talking
